recipes/vs2008_express_vc_python_patch/post-link.bat

```diff
@@ -0,0 +1,2 @@
+call %PREFIX%\Scripts\setup_x64.bat
+copy "C:\Program Files (x86)\Microsoft Visual Studio 9.0\VC\bin\vcvars64.bat" "C:\Program Files (x86)\Microsoft Visual Studio 9.0\VC\bin\amd64\vcvarsamd64.bat"
```

Comment:
I'd much prefer explicit to implicit given what this is doing. Could we please make this a proper script that one calls explicitly?